lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -726,12 +737,16 @@ ifneq "$(SWIFT_WMO)" "1"
 endif
 	@echo "### Compiling" $^
 	@echo "### Swift driver expanded incovation will be:"
+	SWIFT_DRIVER_CLANG_EXEC=$(SWIFT_DRIVER_CLANG_EXEC) \
+	SWIFT_DRIVER_CLANGXX_EXEC=$(SWIFT_DRIVER_CLANGXX_EXEC) \
```

Comment:
Is there no equivalent compiler flag we could add to SWIFTFLAGS instead?

Reply:
No, these overrides are only controlled with an environment variable.
CC @artemcm

Reply:
@artemcm How is this supposed to work in e.g., a CMake build environment then?

Reply:
@etcwilde Could you rework this patch to add `-tools-directory` to SWIFTFLAGS with += update?

Reply:
Watching for https://ci.swift.org/job/swift-PR-Linux/15051/ to pass. If it does, we can set the `-tools-directory`.

Reply:
I have to put this back on the old branch so that I can actually test it

Reply:
Alright, it looks like that passed https://ci.swift.org/job/swift-PR-Linux/15057.

Reply:
Done